mobile/src/main.jsx

```diff
@@ -22,6 +23,8 @@ import Settings from './containers/Settings'
 import { loadState, saveState } from './lib/localStorage'
 import { init } from './lib/cozy-helper'
 
+Raven.config('https://29bd1255b6d544a1b65435a634c9ff67@sentry.cozycloud.cc/2').install()
```

Comment:
@enguerran does the token in the URL always the same, or may it change during runtime?